python/api/requirements.txt

```diff
@@ -1,13 +1,33 @@
-# Note this should be synced with the environment.yml so that we can also run ingestion code on the server. 
-
```

Comment:
Update: I got the declarative_app.py running by using my local Python install and pip-installing just the Flask-Restless and Flask-SqlAlchemy. But, I couldn't get the Docker Python Sandbox to create the environment with these new dependencies added. @jasonrhaas you added the sandbox in #245 - do you know how to get Docker to recreate the environment using an updated requirements.txt located in the root /python folder? We'll want to get that resolved before releasing this to the other developers or we'll have lots of system config questions pouring in.